templates/UmbracoProject/.template.config/template.json

```diff
@@ -82,6 +88,13 @@
       "datatype": "bool",
       "defaultValue": "false"
     },
+    "MinimalHostingModel": {
+      "displayName": "Minimal hosting model",
+      "description": "Replaces Program.cs and Startup.cs with a single simpler Program.cs.",
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false"
```

Comment:
I think this default value should bee true for new installs. We want people to be using the latest stuff after all